apps/rush-lib/src/schemas/version-policies.schema.json

```diff
@@ -103,6 +107,10 @@
         "exemptFromRushChange": {
           "description": "If true, the version policy will not require changelog files.",
           "type": "boolean"
+        },
+        "includeEmailInChange": {
+          "description": "If true, the version policy will request users email on rush change.",
```

Comment:
I think the "email feature" (which we know is a little half-baked at the moment) is really that _changes in the change file have email addresses_ -- being prompted is a side-effect (and I think it's avoidable if you specify the right flags on the command line).

So I'd suggest something more like that -- `If true, generated changelog files will include the author's email address.`.